### ISSUE TITLE
[Feature Request] Steam Deck - equalizer with settings for different audio outputs

### ISSUE THREAD

**Issue by RickS-C137** [ValveSoftware/SteamOS#1310]
This is a feature request for implementing an equaliser to tune the sound output of the Steam Deck. When using headphones or a home theatre system the sound is lacking bass, in my case. Considering that there are a lot of different headphone models etc. out there, which all have different sound characteristics, it would be good to be able to tune the sound/the frequencies to produce an optimal sound stage.
Ideally it's possible to define dedicated settings for the internal speakers, the headphone jack and output via USB-C/HDMI connected to the deck. So the user does not need to adjust the equaliser every time a different output is used.

**Comment by emcy849** [comment]
You can install Easy Effects from the discover store which will give you what you want. Not sure if itll only work in desktop mode though,

**Comment by RickS-C137** [comment]
Yeah, I've read on Reddit about this. There were several people stating that Easy Effects stopped working with SteamOS 3.5 and even before it was buggy. So, in my point of view, no real alternative. Also you would need to change the presets manually, when connecting headphones etc. 
I think it should be possible to detect which output is used for sound and load the preset configured for it.

**Comment by emcy849** [comment]
Oh ok. Another thing 3.5 broke.

**Comment by Rinyuu** [comment]
This feature would be a godsend. I had a custom pipewire EQ configuration file running before, but 3.5 changed some audio related things as well, breaking the routing. Frankly this made my Deck less fun to play ;(! 

Some EQ makes the Deck sound so much better; I'm surprised they didn't ship it with a pre-configured EQ. 

I have also been looking around for any Decky EQ plugins, but they don't seem to exist yet.

**Comment by 27xcb9s9rp-debug** [comment]
Has this been resolved ? Must be a script to modify existing eq in steamOS???
